modules.d/95udev-rules/module-setup.sh

```diff
@@ -93,7 +93,12 @@ install() {
         "${udevdir}"/usb_id \
         "${udevdir}"/v4l_id
 
-    inst_libdir_file "libnss_files*"
+    if ! dracut_module_included "systemd"; then
```

Comment:
If the change removed from systemd-udevd, testing for ! dracut_module_included "systemd" not needed here.